posix.mak

```diff
@@ -182,6 +182,7 @@ STD_DDOC_PRE=$(addsuffix .ddoc, macros html dlang.org ${GENERATED}/${LATEST} std
 SPEC_DDOC=${DDOC} spec/spec.ddoc
 CHANGELOG_DDOC=${DDOC} changelog/changelog.ddoc $(NODATETIME)
 CHANGELOG_PRE_DDOC=${CHANGELOG_DDOC} changelog/prerelease.ddoc
+CHANGELOG_PENDING_DDOC=${CHANGELOG_DDOC} changelog/pending.ddoc
```

Comment:
Would be good to start maintaining some documentation of how the changelog is built. A description or link to one here and the .ddoc files would be nice.

Reply:
Done. I put the description in the respective changelog section